### PR TITLE
fix: remove bad advice

### PR DESCRIPTION
**Pull request by rayng86**
## Changes

- Remove inaccurate statement and fix formatting. Tables are still used for displaying tabular data, but they are no longer the basic building block of the modern web for page layouts.

## Checklist
Check all the boxes that apply to this pull request
- [x] Spelling and Formatting have been checked
- [x] Example code has been included where needed
- [x] References and links are included where needed
- [x] This pull request requires an update to dependencies
- [x] This change looks good on mobile

## How To Review
Describe how the reviewer can see these changes, either by running the code locally on their own machine or viewing it in a published environment.

## Tickets Closed
List tickets closed by this pull request

- Closes [THINKDOCS-##](#link#)

### CODE REVIEW

**Review by THoj13 — COMMENTED** [review]
This diff includes some unrelated formatting changes. Could those formatting changes be removed so that this just includes removed text?

**Comment by rayng86** [comment]
> This diff includes some unrelated formatting changes. Could those formatting changes be removed so that this just includes removed text?

it was auto formatted when i made these changes in the code editor. i don't see issues with formatting fixes as it removes unnecessary whitespaces. no copy was changed except the piece that needed to be removed.

File: content/html/programming-principles/index.md [hunk 1]
```diff
@@ -363,12 +363,6 @@ A title attribute may be added to confer additional meaning, but it is not recom
 
 #### Tables
 
->If you can't enable it, table it!
-
-The HTML Table is the basic building block of the modern web. From replicating elaborate spreadsheets to drawing buttons as tiny 3x3-cell tables where each of the exterior cells represents one corner or side of the button's outline and the text goes in the middle, HTML tables are there to do the "heavy lifting" of page layout. If you want the tl;dr of web development, try `<tr><td>`.
-
-Or you can approach it this way:
-
```
Comment (rayng86) [inline content/html/programming-principles/index.md:371]: begone! I believe this paragraph is what needed to be removed.

**Review by think-luke — APPROVED** [review]
This looks good - [When should you NOT use HTML tables?](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics#when_should_you_not_use_html_tables). 

Like you mentioned, HTML tables are no longer the building blocks of the modern web.

It looks like you have a good answer to Tim's comment and removing unnecessary whitespaces is reasonable here.

**Comment by THoj13** [comment]
>it was auto formatted when i made these changes in the code editor. i don't see issues with formatting fixes as it removes unnecessary whitespaces. no copy was changed except the piece that needed to be removed.

I totally agree with the formatting changes and understand they're results of your code editor auto formatting. My comment was more with respect to those changes being included in the commit of a separate change. It's probably nit picky in this case, but I think separating the formatting changes and copy change to separate commits creates git history that more accurately describes each change. That way if one were to check the history of a line of code that had only had whitespace removed, the latest commit message would refer to a formatting change and not a copy change.

All that said, I do agree with the overall changes of this PR 😄 ✅

**Review by THoj13 — APPROVED** [review]

**Pull request merged**